server/custom-model.sh

```diff
@@ -0,0 +1,9 @@
+cf create-service-key visual-recognition-wws waste-sorter
+API_KEY=$(cf service-key visual-recognition-wws waste-sorter | awk ' /api_key/ {print $2;exit}' | tr -d "\",")
+if [ ${#API_KEY} -eq 0 ]; then
+	echo 'Cannot retrieve your visual recognition api-key, exiting now.'
+	exit 1
+fi
+curl -X POST -F "Landfill_positive_examples=@resources/Landfill.zip" -F "Recycle_positive_examples=@resources/Recycle.zip" -F "Compost_positive_examples=@resources/Compost.zip" -F "negative_examples=@resources/Negative.zip" -F "name=waste" "https://gateway-a.watsonplatform.net/visual-recognition/api/v3/classifiers?api_key=$API_KEY&version=2016-05-20"
```

Comment:
minor suggestion for another patch: the files should be lowercase to avoid case issues on windows/linux/mac platforms